src/api/Governance.ts

```diff
@@ -206,6 +207,16 @@ export class Governance extends API {
     return result.data
   }
 
+  async subscribeToNewsletter(email: string) {
```

Comment:
This featue shouldn't need and extra endpoint, can use the `/subscribe` endpoint directly